app/src/main/java/ch/epfl/sdp/blindly/match/CardStackAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.sdp.blindly.match
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import ch.epfl.sdp.blindly.R
+
+class CardStackAdapter(
+        private var profiles: List<Profile> = emptyList()
```

Comment:
Remove private there since you created setter and getter, so that you can delete both functions.